extensions/binding/org.eclipse.smarthome.binding.digitalstrom/README.md

```diff
@@ -15,71 +15,67 @@ The digitalSTROM-Server is required for accessing any other digitalSTROM-Devices
-
-## digitalSTROM-Scenes
-
-### General
-
-The device scene configuration will be saved persistently to the things to update the device state faster. For that each scene configuration of each device has to read out first, because of the digitalSTROM-rule 9 that requires a waiting period of one minute, that take some time so that at the first start a scene call can take some time to read it out and update the device state. To read it out faster only the discovered or called scenes will be read out. 
- 
-**Note:**
-Because the digitalSTROM-Server can't inform the binding about save scene events at this time, the persistently saved scene configurations can't be updated. The current troubleshooting to read out the new scene configuration after a save scene action at the digitalSTROM-Server is the following:
+Discoverable scenes are all user named scenes, group scenes that are reachable by local push-buttons, zone scenes and apartment scenes. The discovery also will discover all called scenes, if they aren't automatically discovered yet. Temperature control scenes, like *eco* will be ignored, so they can not be discoverd. 
 
-1. delete the thing to delete the persistently saved scene configuration
-2. restart the server to delete the temporary saved scene configuration 
-3. add the thing again to read out the scene configuration again.   
+If you only want to discover one of the thing types, you can start a discovery scan on the thing type you wish have discovered. You can use the command line command, e.g.: ``smarthome:discovery start digitalstrom:namedScene`` to start the scan. What thing types this binding supports please have a look at **Supported Things**. 
 
-## Thing Configuration
+## Thing Configuration and Properties
 
 ### digitalSTROM-Server
 
 The digitalSTROM-Server thing has the following configuration parameter groups: *Connection configuration*, *Server information* and *General configurations*.
 
 #### Connection configuration
 
-If the digitalSTROM-Server isn’t found automatically, e.g. because the server isn’t placed at the local network or the mDNS-service is deactivated, you have to insert the network address or URL and the authentication data manually through the graphical user interface or type it into the \*.thing with textual configuration.
+If the digitalSTROM-Server isn’t found automatically, e.g. because the server isn’t placed at the local network or the mDNS-service is deactivated, you have to insert the network address or URL and the authentication data manually through the graphical user interface or type it into the \*.thing with textual configuration. If you use for authentication the user name and password, the application-token will be automatically get from the digitalSTROM-Server, if their is already a token for this application, otherwise a new application-token will be generated.   
 
```

Comment:
If you use the user name and password for authentication ande there already is a token for this application, it will be automatically retreived from the digitalSTROM-Server, otherwise a new application-token will be generated.